src/env/args.js

```diff
@@ -2,7 +2,13 @@ require('dotenv').config();
 
 const minimist = require('minimist');
 
-const args = minimist(process.argv.slice(2), {
+const runner = process.argv[1].split('/').slice(-1)[0]
```

Comment:
Can you add a comment with an example, of how such argv arguments look like and what it takes as a runner variable?